tests/unit/Jadu/Pulsar/Twig/Macro/Fixtures/elem/paragraph.html.twig

```diff
@@ -1,5 +1,8 @@
-{% import '@pulsar/pulsar/v2/helpers/elem.html.twig' as elem %}
-{{ elem.paragraph(
-    { 'value': 'hello' }
-)
+{% extends '@cssTests/visual-regression-layout.html.twig' %}
+{% block body %}
+{{
+    elem.paragraph(
+        { 'value': 'hello' }
```

Comment:
Could this be formatted in the same way as other macros, with the curly braces on the same line / next to the parentheses?